lition/adapter.js

```diff
@@ -0,0 +1,37 @@
+const { Requester, Validator } = require('@chainlink/external-adapter')
+
+const customError = (data) => data.Response === 'Error'
+
+const customParams = {
+  source: ['source'],
+  date: false,
+  hour: false,
+}
+
+const execute = (input, callback) => {
+  const validator = new Validator(input, customParams)
+  if (validator.error) return callback(validator.error.statusCode, validator.error)
+
+  const jobRunID = validator.validated.id
+  const source = validator.validated.data.source
+  const currentTime = new Date()
+  const date =
+    validator.validated.data.date ||
+    `${currentTime.getUTCFullYear()}-${currentTime.getUTCMonth()}-${currentTime.getUTCDate()}`
+  const hour = validator.validated.data.hour || currentTime.getUTCHours()
```

Comment:
Same thing here... Looks like the API does not return for hour 0 or > 23.

Reply:
I'm able to hit this endpoint: 
https://staking.lition.io/api/v1/energy/source/3/date/2020-10-29/hour/0

Reply:
Yeah, works for me now. Did not work before..